## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 449887`
Commit: [`e4d8b848cc0d2d769a863a4737327f58adcdf006`](https://github.com/NixOS/nixpkgs/commit/e4d8b848cc0d2d769a863a4737327f58adcdf006) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/e4d8b848cc0d2d769a863a4737327f58adcdf006..pull/449887/head))
Merge: [`6c77902af5b7ca182ee07d7f7998cb79c8bcf5d1`](https://github.com/NixOS/nixpkgs/commit/6c77902af5b7ca182ee07d7f7998cb79c8bcf5d1)

Logs: https://github.com/keenanweaver/nixpkgs-review-gha/actions/runs/18344039109


---
### `x86_64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>openjk</li>
    <li>openjk.openja</li>
    <li>openjk.openjo</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>openjk</summary>
<pre>unpacking source archive /nix/store/h62y4iinaaz8cqfb47lx3wrsb1pbdf6c-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/share/doc/openjk -DCMAKE_INSTALL_INFODIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/include -DCMAKE_INSTALL_SBINDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DBuildJK2SPEngine:BOOL=ON -DBuildJK2SPGame:BOOL=ON -DBuildJK2SPRdVanilla:BOOL=ON -DCMAKE_INSTALL_PREFIX=/nix/store/vsvnqkmva74mgmb77v3xv36fih8bh43n-openjk-0-unstable-2025-05-23/opt
CMake Error at CMakeLists.txt:19 (cmake_minimum_required):
  Compatibility with CMake &lt; 3.5 has been removed from CMake.

  Update the VERSION argument &lt;min&gt; value.  Or, use the &lt;min&gt;...&lt;max&gt; syntax
  to tell CMake that the project requires at least &lt;min&gt; but has been updated
  to work with policies introduced by &lt;max&gt; or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>openjk</li>
    <li>openjk.openja</li>
    <li>openjk.openjo</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>openjk</summary>
<pre>unpacking source archive /nix/store/h62y4iinaaz8cqfb47lx3wrsb1pbdf6c-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/share/doc/openjk -DCMAKE_INSTALL_INFODIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/include -DCMAKE_INSTALL_SBINDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DBuildJK2SPEngine:BOOL=ON -DBuildJK2SPGame:BOOL=ON -DBuildJK2SPRdVanilla:BOOL=ON -DCMAKE_INSTALL_PREFIX=/nix/store/sfmdv4mmjlpvqqbhpswq8drxfw404dkm-openjk-0-unstable-2025-05-23/opt
CMake Error at CMakeLists.txt:19 (cmake_minimum_required):
  Compatibility with CMake &lt; 3.5 has been removed from CMake.

  Update the VERSION argument &lt;min&gt; value.  Or, use the &lt;min&gt;...&lt;max&gt; syntax
  to tell CMake that the project requires at least &lt;min&gt; but has been updated
  to work with policies introduced by &lt;max&gt; or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

---
### `x86_64-darwin`
:white_check_mark: *No rebuilds*

---
### `aarch64-darwin`
:white_check_mark: *No rebuilds*

--- a/pkgs/by-name/op/openjk/package.nix
+++ b/pkgs/by-name/op/openjk/package.nix
@@ -45,13 +45,13 @@ let
 in
 stdenv.mkDerivation {
   pname = "openjk";
-  version = "0-unstable-2024-04-07";
+  version = "0-unstable-2025-10-09";
 
   src = fetchFromGitHub {
     owner = "JACoders";
     repo = "OpenJK";
-    rev = "2815211a87ccb8de1b0ee090d429a63f47e0ac7a";
-    hash = "sha256-F3JF6jFgyMinIZ7dZAJ0ugGrJFianh2b6dX5A4iEnns=";
+    rev = "d1cb662f07dfa4c1999edfb5c1a86fd1c6285372";
+    hash = "sha256-XTGe/V4FnQSQA9fY6MmpECs1f2PPk+yTZkAL93UoH/I=";
   };
 
   dontAddPrefix = true;

